### DISCUSSION TITLE
请问如何多台主机同时进行P图和耕种，并连接到同一个主节点？

### DISCUSSION THREAD

**Discussion by hellowind777** [Chia-Network/chia-blockchain#3029] [Q&A]
请问如何多台主机同时进行P图和耕种，并连接到同一个主节点？
我看官方文档，只阐述了主节点+收割机如何工作，并没有说明收割机是否还可以继续P图。
目前我有两台主机，打算第一个主机当主节点，进行P图和耕种，然后第二个主机继续P图和耕种，并连入主节点（目前我使用WIN-GUI客户端），请问如何实现？
有没有大佬可以详细的解答下！诚心邀请：）

引用官方文档部分说明：

`If you are running the GUI and want to run multiple Harvesters

Shut down Chia on main computer

Find your IP address on computer

Make a copy of your main machine CA directory located in c:\users\(your user name)\.chia\mainnet\config\ssl - copy the CA file; you can share the ssl/ca directory on a network drive, USB key, or do a network copy to each harvester. You must copy the new ssl/ca directory with each version of chia-blockchain-- copy the CA file to the harvester machine -- know its location

In new Harvester - follow steps below

Load Chia and use your regular 24 word mnemonic key to see that it works. Then shut down Chia

In c:\users(your user name).chia\mainnet\config file-- open it with notepad

Change enable_upnp: true-- change that to false

Locate harvester: farmer_peer: host: localhost-- change only this location-- type in your main pc ip address (ex 192.192.x.x)

Locate the CA folder you copied from main computer-- know its network location.

Go to command prompt. Type in or copy *cd C:\Users(your username)\AppData\Local\Chia-Blockchain\app-1.1.1\resources\app.asar.unpacked\daemon*

Make sure the (app-1.1.1) is the current version-- this is when version 1.1.1 is active

Run chia init -c [directory] on your harvester, where [directory] is the copy of your main machine CA directory and its network location. This command creates a new certificate signed by your main machine's CA.

[directory] this is where you type the link to where your CA folder is stored-- if on the c drive then type for example c:\ca. The full line would look like chia init -c c:\ca

Then press enter. Once that process is complete *Start both your main pc and the new harvester

The new harvester may take a 10-20 minutes to start the sync procees- it will be a little bit slower- but should start to sync and will make a full copy of the blockchain to get to normal sync. You can create plots on that machine or copy plots over. It will only farm once full sync is completed.`

**Comment by anderzheng** [comment]
按照文档可以的。

**Reply by hellowind777** [reply]
按照官方文档，现在收割机在运行。但主节点电脑看不到收割机的地块。是不是只有收割机爆块了主节点才能看到爆块信息，其他什么都看不到？

**Reply by keliew** [reply]
It'll be there in the future version. You can see the PR, it's there.

**Comment by pangchia** [comment]
I have same question

**Comment by darwinjobchia** [comment]
The new harvester may take a 10-20 minutes to start the sync process- it will be a little bit slower- but should start to sync and will make a full copy of the blockchain to get to normal sync. You can create plots on that machine or copy plots over. It will only farm once full sync is completed.

To know its working

On your main pc under Farm tab- at the bottom select "Hide Advanced Options"- scroll down and " Your Harvester Network" wil now show (2) Node ID-- (1) your main pc and (2) your harvester
also under farm tab under "Last Attempted Proof" your qty of plots on your harvester will also show up there

**Reply by darwinjobchia** [reply]
官方文档says ...you can create plots on that machine or copy plots over

**Reply by keliew** [reply]
new version doesn't require full sync before farming...
